src/APIStatus/APIStatus.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState, useEffect } from "react";
+import { MenuItem, Typography, Button } from "@material-ui/core";
+import { CheckCircleRounded } from "@material-ui/icons";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import ErrorIcon from "@material-ui/icons/Error";
+import axios from "axios";
+import { apiCorsUrl, APIURL } from "../Devices/hologramConstants";
+import QueryString from "qs";
+import { apiPassword, apiUsername, onfarmAPI } from "../utils/api_secret";
+const hologramAPI = "/api/1/users/me";
+
+const ApiStatus = () => {
+  const [devAPI, setDevAPI] = useState(null);
+  const [gitHubAPI, setGitHubAPI] = useState(null);
+  useEffect(() => {
+    fetchfromInternal();
+    fetchFromAPIHologram();
+    fetchFromPHPAPI();
+    fetchFromAPIWeather(onfarmAPI);
```

Comment:
naming this function as weather API is misleading. OnFarm API is different from the weather API